distribution/lib/Standard/Base/0.0.0-dev/src/Data/Enso_Cloud/Utils.enso

```diff
@@ -1,3 +1,4 @@
+private
```

Comment:
@JaroslavTulach per your suggestion, I thought that it may make sense to mark this module as `private`, so I did!

It only required moving `as_hideable_value` elsewhere (but I assume the new place actually fits it better anyway), because it needs to be usable in other libraries (e.g. `AWS`).